src/main/python/pre_setup.py

```diff
@@ -39,7 +39,7 @@
     shutil.rmtree(TMP_DIR, True)
 os.mkdir(TMP_DIR)
 
-SYSTEMDS_BIN = 'systemds-*-SNAPSHOT-bin.zip'
+SYSTEMDS_BIN = 'systemds-*-bin.zip'
```

Comment:
This change leads to test failure. That is why I suggested using `sed `to do it only during release.